/**
 * Api Messages
 */
syntax = "proto3";

package api;

//import "google/protobuf/any.proto";
//import "google/protobuf/timestamp.proto";

message Envelope {
  oneof message {
    Hello hello = 1;
  }
}

message Hello {
  string message = 1;
}

message Indices {
  string index = 1;
  string symbol = 2;
  string key = 3;
}
message Stocks {
  string name = 1;
  string symbol = 2;
  string key = 3;
}

message ListIndicesReply {
  repeated Indices indices = 1;
}

message ListIndicesStocksRequest {
  string symbol = 1;
}

message ListIndicesStocksReply {
  repeated Stocks stocks = 1;
}